src/commands/remoteDebug/getRemoteDebugLanguage.ts

```diff
@@ -6,15 +6,20 @@
 import { SiteConfig } from '@azure/arm-appservice';
 import { RemoteDebugLanguage } from '@microsoft/vscode-azext-azureappservice';
 
-export function getRemoteDebugLanguage(siteConfig: SiteConfig): RemoteDebugLanguage {
+export function getRemoteDebugLanguage(siteConfig: SiteConfig, appServicePlan?: string): RemoteDebugLanguage {
     // Read siteConfig.linuxFxVersion to determine debugging support
     //   If the Function App is running on Windows, it will be empty
-    //   If the Function App is running on Linux consumption, it will be empty
+    //   If the Function App is running on Linux consumption plan remote debugging is not supported
     //   If the Function App is running on a Linux App Service plan, it will contain Docker registry information, e.g. "DOCKER|repo.azurecr.io/image:tag"
-    //      The blessed Node.js Docker image will be something like "DOCKER|mcr.microsoft.com/azure-functions/node:2.0-node8-appservice"
-    if (siteConfig.linuxFxVersion && siteConfig.linuxFxVersion.startsWith('DOCKER|mcr.microsoft.com/azure-functions/node')) {
-        return RemoteDebugLanguage.Node;
+    //      or it will contain language information, e.g. "Node|14"
+    if (appServicePlan) {
+        if (/^[^yY]$/.test(appServicePlan)) {
+            if (siteConfig.linuxFxVersion && (siteConfig.linuxFxVersion.startsWith('DOCKER|mcr.microsoft.com/azure-functions/node') || siteConfig.linuxFxVersion.toLowerCase().startsWith('node|'))) {
```

Comment:
lmao I actually meant _this_ line should be used as a regex, not the consumption plan check 🤣😂😹

Reply:
I actually preferred your original way of checking for consumption plans. 😅

Reply:
Lol I was confused for a second why you would want it for the consumption plan 😄, but I will change that back and put a regex for the other line.